js/ftx.js

```diff
@@ -2283,4 +2285,48 @@ module.exports = class ftx extends Exchange {
         const result = this.safeValue (response, 'result', {});
         return this.parseFundingRate (result, market);
     }
+
+    async fetchBorrowRates (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetSpotMarginBorrowRates ();
+        const timestamp = this.milliseconds ();
+        const result = this.safeValue (response, 'result');
+        const rates = [];
+        for (let i = 0; i < result.length; i++) {
+            const rate = result[i];
+            const currency = this.safeCurrencyCode (this.safeString (rate, 'coin'));
+            rates[currency] = {
+                'currency': currency,
+                'rate': this.safeNumber (rate, 'previous'),
+                'period': 3600000,
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'info': rate,
+            };
+        }
+        return rates;
+    }
+
+    async fetchBorrowRate (code, params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetSpotMarginBorrowRates (params);
+        const timestamp = this.milliseconds ();
+        const currencyId = this.currency (code)['id'];
+        const result = this.safeValue (response, 'result');
+        for (let i = 0; i < result.length; i++) {
+            const rate = result[i];
+            const coin = this.safeCurrencyCode (this.safeString (rate, 'coin'));
+            if (coin === currencyId) {
```

Comment:
This line is wrong – we should never compare unified currency codes to exchange-specific currency ids for equality.